### PR TITLE
added meta-packages

### PR DESCRIPTION
**Pull request by meets2tarun**
Added metapackage which include vlc,lximage-qt,krita.
[issue](https://github.com/fossasia/meilix/issues/3)

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ datafiles="image-${arch}.tar.lzma sources.list"
 # Necessary development tool packages to be installed on build host
 devtools="debootstrap genisoimage p7zip-full squashfs-tools ubuntu-dev-tools"
 
-# Make sure we have the data files we need
+# Make sure we have the data files we needo
```
Comment (Abhi2424shek) [inline build.sh]: Typo

**Comment by meets2tarun** [comment]
I corrected the typo error.

**Comment by meets2tarun** [comment]
Please checkmark the solved points in [issue](https://github.com/fossasia/meilix/issues/3)

**Comment by Abhi2424shek** [comment]
Thanks I did, please check the issue ;)

**Pull request merged**